kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -281,16 +281,25 @@
                     </template>
                     <template #content>
                       <div
+                        v-if="isItemExpanded(question.question_id)"
                         :id="`question-panel-${question.question_id}`"
                         :ref="`question-panel-${question.question_id}`"
                         :style="{ userSelect: dragActive ? 'none!important' : 'text' }"
                       >
-                        <p
-                          v-if="isItemExpanded(question.question_id) && !dragActive"
-                          class="question-content-panel"
-                        >
-                          CONTENT OF {{ question.title }}
-                        </p>
+                        <ContentRenderer
```

Comment:
Should probably use the `:interactive=false` prop here too: https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/views/ContentRenderer/mixin.js#L135